src/table/src/TableVirtualBody.js

```diff
@@ -52,7 +53,7 @@ const TableVirtualBody = memo(function TableVirtualBody(props) {
     }
 
     // When height is still 0 (or paneRef is not valid) try recursively until success.
-    requestAnimationFrame(() => {
+    requestId.current = requestAnimationFrame(() => {
```

Comment:
Good catch! Should we also capture the animation frame request id for the call on line 136? I'm thinking that also has potential for a memory leak.

Reply:
Yes, as you might know, frameRequests should always be cancelled once the component is destroyed/unmounted. The scenario file I executed might have missed covering the call of the enclosing function of line 136, hence was not pointed out in Memlab leaks result, but best to clear that too :)